.github/workflows/build_artifacts.yml

```diff
@@ -45,6 +45,9 @@ jobs:
                   MDIXColorsVersion: ${{ inputs.mdix-colors-version }}
                   MDIXMahAppsVersion: ${{ inputs.mdix-mahapps-version }}
 
+            - name: Generate Version File
+              run: echo ${{ inputs.mdix-version }} > src/MainDemo.Wpf/version.txt
+
```

Comment:
This feels kinda hacky, but I unfortunately can't suggest a better way because I don't have much experience with pipelines.